lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/ashrae_90_1_prm_2019_WaterHeaterMixed.rb

```diff
@@ -0,0 +1,126 @@
+class ASHRAE901PRM2019 < ASHRAE901PRM
+
+  # Apply the prm parameter to a water heater based on the
+  # building area type.
+  # @param water_heater_mixed [OpenStudio::Model::WaterHeaterMixed] water heater mixed object
+  # @param building_type [String] the building type (For consistency with the standard class, not used in the method)
+  # @param swh_building_type [String] the swh building are type
+  # @return [Boolean] returns true if successful, false if not
+  def model_apply_water_heater_prm_parameter(water_heater_mixed, building_type_swh)
+    # get number of water heaters
+    if water_heater_mixed.additionalProperties.getFeatureAsInteger('component_quantity').is_initialized
+      comp_qty = water_heater_mixed.additionalProperties.getFeatureAsInteger('component_quantity').get
+    else
+      comp_qty = 1
+    end
+
+    # Get the capacity of the water heater
+    capacity_w = water_heater_mixed.heaterMaximumCapacity
+    if capacity_w.empty?
+      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.WaterHeaterMixed', "For #{water_heater_mixed.name}, cannot find capacity, standard will not be applied.")
```

Comment:
To avoid duplication, use the existing water_heater_mixed_apply_efficiency function in Standards.WaterHeaterMixed.rb instead of developing a new function.